src/DependenciesPackager/CrossgenContext.cs

```diff
@@ -179,13 +179,14 @@ private bool CrossgenAssembly(string publishFolder, PackageEntry package, Librar
             return exitCode == 0;
         }
 
-        private DirectoryInfo CreateSubDirectory(string cacheBasePath, PackageEntry package, LibraryAsset asset)
+        private DirectoryInfo CreateSubDirectoryInLowerCase(string cacheBasePath, PackageEntry package, LibraryAsset asset)
         {
             var subDirectoryPath = Path.Combine(
                 cacheBasePath,
                 package.Library.Identity.Name,
                 package.Library.Identity.Version.ToNormalizedString(),
-                Path.GetDirectoryName(asset.RelativePath));
```

Comment:
Is this resolve available in the 3.5.0-beta2? Because it is used by project model: https://github.com/dotnet/cli/blob/rel/1.0.0-preview2/src/Microsoft.DotNet.ProjectModel/project.json#L12

I try to avoid upgrading the dependencies on CLI sdk.